src/widget/wtrackproperty.cpp

```diff
@@ -84,6 +84,18 @@ void WTrackProperty::mouseMoveEvent(QMouseEvent *event) {
     }
 }
 
+void WTrackProperty::mousePressEvent(QMouseEvent* event) {
+    if ((event->buttons() & Qt::LeftButton) && m_pCurrentTrack) {
+        if (!m_clickTimer.isActive()) {
+            m_clickTimer.setSingleShot(true);
+            m_clickTimer.start(400);
```

Comment:
I am not very experienced with Qt yet but can we use something like the standard `mouseDoubleClickEvent` instead of a custom timer?

Reply:
yep, right!
Benefit: OS double-click period applies.

Reply:
The code looks much cleaner and easier to understand now. Thanks for the changes.